master.js

```diff
@@ -95,19 +77,24 @@ $(function() {
     $('tr.category>td'  ).prop('colSpan', $('tr.supertest:first>td:visible').length);
   };
 
-
-
-  table.on("floatThead", function (evt, isFloating, $container) {
-    $container.toggleClass("floating-header", isFloating);
-  });
-
+  /** Sticky header */
+  var platformTypesHeader = document.querySelector(
```

Comment:
Since it will not work in ancient engines, maybe it would be better to use jQuery like in other parts of this script?

Reply:
I don't think there are browsers out there which don't support this. The latest browser that doesn't is IE7: https://caniuse.com/#feat=queryselector

And since compat-table is primarily a developer-facing tool, I believe we don't have to support this.

The only argument for using jquery here would be consistency, but since this piece of code is quite isolated, I don't think it's an issue. What do you think?

Reply:
I think that it would be better to use jQuery because of both reasons.

Reply:
I agree; jquery plugins are best avoided, but jquery itself is quite useful.

Reply:
🤷‍♀